drf_spectacular/contrib/authentication.py

```diff
@@ -3,7 +3,7 @@
 
 class SimpleJWTScheme(OpenApiAuthenticationExtension):
     target_class = 'rest_framework_simplejwt.authentication.JWTAuthentication'
-    name = 'jwtAuth'
+    name = 'simplejwt'
```

Comment:
happy to remove this if requested.  I realise it would change the generated schema for existing users, which is technically a breaking change.  IMO better to get these names 'canonicalised' sooner than later.  There are many other jwt handlers, including newer ones like drf-jwt-knox

Reply:
yes, i prefer it the other way for now. i don't like exposing the used lib through the name. they all should implement the jwt standard. regarding naming collisions, i would assume that one chooses one lib and then sticks with it. why would anyone use 2 or more jwt implementations. if need arises, we can change it later.